src/components/modal/examples/components/FancyModal.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div>
```

Comment:
it would also be great to get an example `role="alertdialog"` modal in here

Reply:
Will follow up in another PR